AnnoyancesFilter/sections/popups.txt

```diff
@@ -95,6 +95,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !
 cybernews.com##.dialog-lightbox-widget
+||livingtraditionally.com/wp-content/cache/min/1/*.js
```

Comment:
this rule is too wide despite of low website traffic.
Not sure if URLs are random.
On my end:
https://livingtraditionally.com/wp-content/cache/min/1/f6d40e07318704badcdcdac0aff1556a.js

the rule for `/gtm-` is acceptable